### ISSUE TITLE
Sortable Grid page accessibility issues

### ISSUE THREAD

**Issue by FayCross** [thexerteproject/xerteonlinetoolkits#1394]
There are various issues with the sortable grid page but these are the accessibility related ones:

- [ ] The layout of the page uses an empty table with content positioned absolutely on top. Screen readers therefore get told about the empty table and the context of where each item is within the table is not clear without seeing the screen.
- [ ] Ticks & crosses are not labelled in a sensible way for screen reader users.
- [ ] Feedback that appears on the screen after submitting correct answers does not get announced to screen reader users via aria-live polite.
- [ ] The hints that are shown as you move labels around (to show which other labels will be moved & where they will be moved to) are not placed correctly if any of the labels are more than one line of text tall. This is a problem when using the mouse or keyboard to move labels.

**Comment by FayCross** [comment]
For info, these issues are noted on the 3.14 accessibility statement:

The Sortable Grid pages in XOT projects require users to rearrange labels within the cells of a table. Although labels can be focused and selected using the keyboard, there is no context available to screen reader users regarding the positioning of the labels within the table. This fails [1.3.1 (Info and Relationships)](https://www.w3.org/WAI/WCAG22/quickref/?showtechniques=144%2C1410%2C211%2C222%2C247#info-and-relationships).
